modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -0,0 +1,24 @@
+= Managing APIs with Anypoint Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, manage APIs
+
+image::manage-service-mesh-breadcrumb.png[75%, 75%]
+
+After you bind your API, the API in API Manager is marked as Active.You are now ready to apply policies to protect your APIs.
+
+image::api-manager-status-for-service-mesh.png[10%.10%]
+
+To apply policies:
+
+. Go to *API Manager > API Administrations > Policies*.
```

Comment:
should we need to link to specific policy documentation?

Reply:
OK!

Reply:
Done!